handlers/challenge_handler.py

```diff
@@ -410,6 +410,16 @@ def execute(self, slack_wrapper, args, channel_id, user_id):
         # Show confirmation message
         slack_wrapper.post_message(channel_id, message)
 
+class ReloadCommand(Command):
+    """Reload the ctf information from slack to reflect updates of channel purposes."""
+
+    def execute(self, slack_wrapper, args, channel_id, user_id):
+        """Execute the Reload command."""
+
+        slack_wrapper.post_message(channel_id, "Start updating CTF challenge handler...")
```

Comment:
Should we start using more user-friendly messages such as : 
> Updating CTFs and challenges...
> Update finished.

Reply:
The reason I ask is because most people don't know/care about the concept of "handlers" :)

Reply:
Sure thing. That's why "localization" should be done by someone other than a developer :)

Will update this later. Maybe we should refactor all the texts out of the code in something like a resource file, separating code and texts.